app/mailers/fixity_failure_mailer.rb

```diff
@@ -0,0 +1,24 @@
+#class FixityFailureMailer < ApplicationMailer
+
+class FixityFailureMailer < ApplicationMailer
+
+  def fixity_failure_email
+    @fixity_check = params[:fixity_check]
```

Comment:
It's been a while since I wrote a mailer -- I'm a bit disturbed by the way the `params` are not documented in a method signature, is this really the standard way to do it? I'm going to double check in the [Rails Action Mailer guide](https://guides.rubyonrails.org/action_mailer_basics.html)...

That does seem to be what the examples there do, okay then. 🤷‍♂   What if we raise if the arg isn't supplied though, to avoid more confusing debugging if it's accidentally missing?

    @fixity_check = params[:fixity_check] || raise ArgumentError.new("Required params[:fixity_check] missing")

Reply:
@eddierubeiz You told me this was ready for review, but you don't seem to have responded to or addressed this comment? Please use the comments on a PR as a checklist of things to consider before asking for another review -- if you don't think one is necessary or is a bad idea, you can say that in the comment thread instead of just leaving it there hanging.

Reply:
I am checking for the presence of params[:fixity_check] and raising an error if it's not present. Just not on the same line (the way you had it resulted in a syntax error)